### DISCUSSION TITLE
Webflow integration clarifications

### DISCUSSION THREAD

**Discussion by davhad** [AntlerVC/firetable#1308] [Support / Q&A] [answered]
Hello, I just created a Webflow sync table using the existing and much-hyped Rowy webflow template and a few things remain unclear to me without any documentation to clarify them.

1. When the Webflow collection id is added, is the rowy table supposed to recreate those fields or do we have to do that manually?

2. When the webflow template is used, is the data already in webflow synced into Rowy? If not do we have to do a manual webflow csv export followed by a csv import to rowy?

3. After rowy table is created and setup, there is one-way sync from rowy to webflow when the publish button is pressed. Is there a reverse sync as well when webflow data is updated?

4. How are the following webflow fields mapped into Rowy? single select, multi-select, and collection relationship?

5. In the presence of images I'm getting an error on oublishing that I don't have permissions to do that. Any suggested fixes?

Thank you

**Comment by davhad** [comment]
Any updates on these questions?

**Comment by BeeBombshell** [comment, marked as answer]
Hey @davhad, here to help!

> 1. When the Webflow collection id is added, is the rowy table supposed to recreate those fields or do we have to do that manually?

When the Webflow collection ID is added, Rowy does not automatically recreate the fields in the table. You'll need to manually create the corresponding fields in Rowy to match the fields in your Webflow collection. This allows you to define the structure and data types that align with your specific requirements. (However, if you're using our Webflow template, each of the Webflow fields is mapped to the columns in our Rowy table as shown in the [YouTube Tutorial](https://www.youtube.com/watch?v=gpZWPQljKiw))

> 2. When the webflow template is used, is the data already in webflow synced into Rowy? If not do we have to do a manual webflow csv export followed by a csv import to rowy?

The template does not automatically sync the existing data from Webflow into Rowy. You will need to manually export the data from Webflow, typically in CSV format, and then import that data into Rowy. For more information on import via CSV, [check out our docs](https://docs.rowy.io/import-export-data/import-csv#autogenerated-id-vs-custom-id).

> 3. After rowy table is created and setup, there is one-way sync from rowy to webflow when the publish button is pressed. Is there a reverse sync as well when webflow data is updated?

By default, Rowy provides one-way sync from Rowy to Webflow. This means that when you make changes or updates in Rowy and press the publish button, those changes will be synced to Webflow. However, there is currently no automatic reverse sync from Webflow to Rowy. If you update your data in Webflow, you would need to manually update the data in Rowy **or you could choose to bulk update fields** ([check out how!](https://docs.rowy.io/import-export-data/import-csv#how-to-bulk-update-if-you-have-autogenerated-ids))

> 4. How are the following webflow fields mapped into Rowy? single select, multi-select, and collection relationship?

**Single select:** Could be mapped to a Short / Long Text or [Single Select field](https://docs.rowy.io/field-types/single-select) in Rowy.
**Multi-select:** Multi-select fields can be mapped to a Short Text / Long Text or [Multi Select field](https://docs.rowy.io/field-types/multi-select) in Rowy.
**Collection relationship:** Webflow's collection relationship fields represent a reference to another collection. In Rowy, you can create a relationship by using a [Reference field type](https://docs.rowy.io/field-types/supported-fields) that references a Firestore document in another table.

> 5. In the presence of images I'm getting an error on oublishing that I don't have permissions to do that. Any suggested fixes?

If you encounter permissions issues related to images when publishing from Rowy to Webflow, you should review your Webflow account settings and ensure that you have the necessary permissions to upload and modify images in Webflow. Perhaps you could share some error message screenshots or video recordings?
